src/lib/wizard/components/Deploy.svelte

```diff
@@ -99,6 +106,8 @@
     if (globalState.contract?.target && compilationResult) {
       inputs = getConstructorInputsWizard(globalState.contract.target, compilationResult.output.contracts);
 
+      globalState.form.constructorArguments.required = inputs.length
```

Comment:
Same as above, lets use setters, this is better to control the global state modifiers from the same file they are declared